src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import { Users } from '../../types/Users';
+import { Comments } from '../../types/Comments';
+import { Posts } from '../../types/Posts';
+
+type Props = {
+  post: Posts
+  users: Users[];
+  comments: Comments[];
+  userId: number;
+};
+
+export const PostInfo: React.FC<Props> = ({
+  post,
+  users,
+  comments,
+  userId,
+}) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        <UserInfo user={users[userId - 1]} />
+      </p>
```

Comment:
Wrap the UserInfo component inside a 'div' or 'span' element, instead of a 'p' element, to avoid potential styling issues.